app/src/main/java/com/DatabaseHandler.java

```diff
@@ -0,0 +1,122 @@
+package com;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.DatabaseUtils;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.util.Log;
+
+import com.simpragma.recipe.recipeapplication1.DataBaseResult;
+import com.simpragma.recipe.recipeapplication1.Result;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by shubhank on 2/4/18.
+ */
+
+public class DatabaseHandler extends SQLiteOpenHelper {
+
+    // All Static variables
+    // Database Version
+    private static final int DATABASE_VERSION = 2;
+
+    // Database Name
+    private static final String DATABASE_NAME = "recipeManager1";
+
+    // Contacts table name
+    private static final String TABLE_RECIPE = "recipeitems1";
+
+    // Contacts Table Columns names
+    private static final String KEY_ID = "id";
+    private static final String KEY_TITLE = "title";
+    private static final String KEY_INGREDIENTS = "ingredients";
+    private static final String KEY_THUMBNAIL = "thumbnail";
+    private static final String KEY_HREF = "href";
+
+
+
+
+    public DatabaseHandler(Context context) {
+        super(context, DATABASE_NAME, null, DATABASE_VERSION);
+    }
+
+    @Override
+    public void onCreate(SQLiteDatabase db) {
+        String CREATE_CONTACTS_TABLE = "CREATE TABLE " + TABLE_RECIPE + "("
+                + KEY_ID + " INTEGER PRIMARY KEY," + KEY_TITLE + " TEXT,"
+                + KEY_INGREDIENTS + " TEXT,"
+                + KEY_HREF + " TEXT,"
+                + KEY_THUMBNAIL + " TEXT" + ")";
+        db.execSQL(CREATE_CONTACTS_TABLE);
+    }
+
+    @Override
+    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
+        db.execSQL("DROP TABLE IF EXISTS " + TABLE_RECIPE);
```

Comment:
It's good to log such things.

Reply:
I mention Log
 Log.d(TAG,"Upgrade method Recipe Table");